pebbles/static/js/directives/NotificationDirective.js

```diff
@@ -28,6 +28,43 @@ app.directive('pbNotifications', ['Restangular', 'AuthService', 'config', functi
             if (AuthService.isAuthenticated()) {
                updateNotifications();
             }
+
+	    /* To send new notifications also as desktop notifications */
+            var stop;
+            scope.startPolling = function() {
```

Comment:
Polling should not be done in a directive

Reply:
Is there a specific reason for this ? Not a fan of polling but this is the most appropriate place to check for new notifications.

Reply:
Can't it be done here: https://github.com/CSCfi/pebbles/blob/master/pebbles/static/js/controllers/DashboardController.js#L266
If there's already a polling process taking place then it's efficient to avoid another one

Reply:
I am afraid not. Dashboard controller has nothing to do with notification. And if we do it there then the user must be in dashboard to receive notification. For example, if he is in accounts page he will not receive any notification.

Reply:
Well then, this has to be the way.